governance-exercises/CivicSage Exercises/Ex01/3.1 Adjust Spork D3M Parameters - JULY 24, 2023.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Adjust Spork D3M Parameters - JULY 24, 2023
+summary: Signal your support or opposition to adjusting the Spork DAI Direct Deposit Module (D3M) parameters.
+discussion_link: https://github.com/makerdao/community/blob/master/governance-exercises/poll-exercise-01.md
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-07-24T16:00:00
+end_date: 2023-07-27T16:00:00
+---
+# Poll: Adjust Spork D3M Parameters - JULY 24, 2023
+
+The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of Spork. The community can vote in this poll to express support or opposition to using the Spork Direct Deposit DAI Module (D3M) with the listed parameters. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, July 24 at 16:00 UTC.
```

Comment:
Good. In this case, is okay to use "Governance Facilitators" without specifying Endgame Edge